packages/connectivity/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.0.2
```

Comment:
For a breaking change we should probably bump to 0.1.0. This avoids breaking people that depend on `^0.0.1`

Reply:
Sounds good! Done.